docker/README.md

```diff
@@ -0,0 +1,82 @@
+# Docker's version of `amc2moodle`
+
+A docker version of `amc2moodle` is available and it start a daemon server to automatically execute `amc2moodle` or `moodle2amc`.
```

Comment:
typo : "it starts"
add something like : Alternatively, it is also possible to run an interactive shell in the docker to recover standard `amc2moodle` command line interface.